Fix test_run_count_subjob_in_x of Test_run_count

shilpakodli opened the pull request:
<!--- Please review your changes in preview mode -->
<!--- Provide a general summary of your changes in the Title above -->

#### Describe Bug or Feature
test_run_count_subjob_in_x  was failing with Job not in 'H' state after execjob_begin hook with reject request is created.
Test is submitting an array of jobs and while second subjob is in running state, creating execjob_begin hook with reject. On few machines, hook create and import was taking little longer time that third subjob would also get finished by the time test checks for 'H' state of third subjob.


#### Describe Your Change
Increased sleep time of Array jobs such that second subjob would run for longer time and third subjob would start executing after hook has imported.


#### Attach Test and Valgrind Logs/Output
[test_run_count_subjob_in_x_before_fix.txt](https://github.com/openpbs/openpbs/files/6675381/test_run_count_subjob_in_x_before_fix.txt)
[test_run_count_subjob_in_x_after_fix.txt](https://github.com/openpbs/openpbs/files/6676426/test_run_count_subjob_in_x_after_fix.txt)



<!--- Pull Request Guidelines: [Pull Request Guidelines](https://pbspro.atlassian.net/wiki/spaces/DG/pages/1187348483/Pull+Request+Guidelines) -->

bhroam commented on test/tests/functional/pbs_test_run_count.py:
```diff
@@ -154,14 +154,14 @@ def test_run_count_subjob_in_x(self):
         # Create an execjob_begin hook that rejects the job
         self.create_reject_begin_hook()
-        time.sleep(8)
+        time.sleep(15)
```
Make both of these sleeps offsets in the expect.  Also consider adding a logger message explaining what you are doing.

shilpakodli commented:
@bhroam I have addressed your review comments, Please review latest code changes.
[test_run_count_subjob_in_x_latest_logs.txt](https://github.com/openpbs/openpbs/files/6691798/test_run_count_subjob_in_x_latest_logs.txt)

bhroam reviewed (approved):
LGTM
